### ISSUE TITLE
WASI support

### ISSUE THREAD

**Issue by mrceperka** [wasmerio/php-ext-wasm#77]
### WASI support
Hello :wave: , quick question :)

According to this article - https://hacks.mozilla.org/2019/03/standardizing-wasi-a-webassembly-system-interface/ one is able to _just run_ WASM binary that was compiled with WASI support (it's possible that I mix terms, sorry)

I've found somewhere on Twitter, that PHP is not yet supported by https://github.com/CraneStation/wasmtime.

But what if I'm gonna use this extension, which uses Wasmer, which has support for WASI, it's gonna work right :pray:?

Thanks for you time.

**Comment by Hywan** [comment]
Yes, we can add support for WASI quite easily. Give me few days to work on it :-)!

**Comment by mrceperka** [comment]
Oh my... You are so awesome! Thanks :pray:

**Comment by Hywan** [comment]
(more than a few weeks actually… working on it soon, see https://github.com/wasmerio/wasmer/pull/1030 to follow latest improvements)

**Comment by passcod** [comment]
What's the status on this, given the above linked PR was merged?
